fitbenchmarking/cli/main.py

```diff
@@ -38,20 +38,27 @@ def get_parser():
 
     epilog = '''Usage Examples:
```

Comment:
The usage examples need to include `-p` in both and I think there should be a bit of text before the examples stating that this is run with the source code downloaded.